config/config-package.py

```diff
@@ -196,6 +196,9 @@ def copy_with_meta(template_name, destination, config_type, **kw):
 fail_under = meta_cfg['coverage'].setdefault('fail-under', 0)
 coverage_run_additional_config = meta_cfg['coverage-run'].get(
     'additional-config', [])
+flake8_additional_sources = meta_cfg['flake8'].get('additional-sources', '')
+if flake8_additional_sources:
+    flake8_additional_sources = ' ' + flake8_additional_sources
```

Comment:
Maybe some comments helps in six months. 
```suggestion
    # Avoid whitespace at  end of line if no additional sources are provided.
    flake8_additional_sources = ' ' + flake8_additional_sources
```

Reply:
Thanks for the suggestion, as I am not able to change it in GitHub and I want to get rid of the additional withe space, I am going to change it locally.